Microsoft.Toolkit.Uwp.UI.Behaviors/Keyboard/KeyDownTriggerBehavior.cs

```diff
@@ -0,0 +1,66 @@
+using System.Windows.Input;
+using Microsoft.Xaml.Interactivity;
+using Windows.System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Behaviors
+{
+    /// <summary>
+    /// This behavior listens to a key down event on the associated <see cref="UIElement"/> when it is loaded and executes an action.
+    /// </summary>
+    [TypeConstraint(typeof(FrameworkElement))]
+    public class KeyDownTriggerBehavior : Trigger<UIElement>
+    {
+
+        /// <summary>
+        /// The DP to store the <see cref="Key"/> property value.
+        /// </summary>
+        public static readonly DependencyProperty KeyProperty = DependencyProperty.Register(
+            "Key",
+            typeof(VirtualKey),
+            typeof(KeyDownTriggerBehavior),
+            new PropertyMetadata(null));
+
+        /// <summary>
+        /// Gets or sets the key to listen when the associated object is loaded.
+        /// </summary>
+        public VirtualKey Key
+        {
+            get => (VirtualKey)GetValue(KeyProperty);
+            set => SetValue(KeyProperty, value);
+        }
+
+        public static readonly DependencyProperty CommandProperty = DependencyProperty.Register(
```

Comment:
Am I missing where the property is for this? We don't need this right?

Reply:
Left over from my old code. Deleted. Good catch!